plugins/clipboard/plugin.js

```diff
@@ -1545,7 +1545,7 @@
 					dragRange = clipboard.dragRange;
 
 				// Do nothing if it was not possible to get drop range.
-				if ( !dropRange ) {
+				if ( !dropRange || !dragRange ) {
```

Comment:
The comment just above could be updated, but I guess we want to keep the diff with the upstream version minimal?

Reply:
Yes indeed, usually we would update the comment too because comments are important for code we might re-read in the future. Here, if this prove to resolve the crash, I doubt we'll re-read that part again in the future. But no strong opinion, if you prefer the comment also, I can. To me the code is quite explicit and to be honest I didn't even read the comment.

Reply:
OK for me then :+1: